File: das_backend/src/main/kotlin/ch/sbb/backend/logging/domain/LogEntry.kt
package ch.sbb.backend.logging.domain

import java.time.OffsetDateTime

data class LogEntry(
    val time: OffsetDateTime,
    val source: String,
    val message: String,
    val level: LogLevel,
    val metadata: Map<String, String>? = emptyMap()
)

File: das_client/logger/lib/src/log_entry.dart
import 'package:logger/src/log_level.dart';

class LogEntry(
  final String message,
  final LogLevel level,
  final Map<String, dynamic> metadata,
) {
  this : time = DateTime.now().millisecondsSinceEpoch / 1000;

  final double time;
}
